### DISCUSSION TITLE
Looping from javascript in Automations (trigger -> js scripting -> add rows)

### DISCUSSION THREAD

**Discussion by polauf** [Budibase/budibase#9559] [Help]
I have simple pipeline:
![image](https://user-images.githubusercontent.com/7090336/217219867-28c970c5-b8fe-48f6-adb0-f83ba9472fb1.png)

How do I make it work?
Tried to feed it simple Array as binding to looping like `[1,2,3]` but its get stuck. (Infinite loop probably or its casting the previous stuff `{{ steps.1.value }}` to string)

**Comment by melohagan** [comment]
Hey @polauf 

What does your **JS Scripting** step look like?

Could you also expand the **Looping** and **Create Row** and share the screenshots of that please
